src/Tribe/Repositories/Attendee/RSVP.php

```diff
@@ -104,4 +105,126 @@ public function checked_in_keys() {
 		return $this->limit_list( $this->key_name, parent::checked_in_keys() );
 	}
 
+	/**
+	 * {@inheritDoc}
+	 */
+	public function update_additional_data( $attendee, $ticket, $attendee_data ) {
+
+		$attendee_data = $this->format_attendee_data( $attendee, $ticket, $attendee_data );
+
+		try {
+			$query = $this->where( 'id', $attendee->ID )
+			              ->set( 'ticket_id', $ticket->ID )
+						  ->set( 'full_name', $attendee_data['full_name'] )
+						  ->set( 'email', $attendee_data['email'] )
+						  ->set( 'event_id', $attendee_data['event_id'] )
+						  ->set( 'security_code', $attendee_data['security_code'] )
+						  ->set( 'order_id', $attendee_data['order_id'] )
+						  ->set( 'optout', $attendee_data['optout'] )
+						  ->set( 'attendee_status', $attendee_data['attendee_status'] )
+						  ->set( 'price_paid', $attendee_data['price_paid'] )
```

Comment:
:warning: Found precision alignment of 2 spaces.

<sub>hash: e7c2a52aa53af16a92fb64b4f36a3637</sub>